framework/src/main/java/org/tron/core/db/Manager.java

```diff
@@ -1049,19 +1040,18 @@ public synchronized void pushBlock(final BlockCapsule block)
       logger.info(SAVE_BLOCK + newBlock);
     }
     //clear ownerAddressSet
-    synchronized (pushTransactionQueue) {
```

Comment:
why delete synchronized (pushTransactionQueue)

Reply:
> why delete synchronized (pushTransactionQueue)
BlockingQueue implementations are thread-safe. All queuing methods are atomic in nature and use internal locks.